### ISSUE TITLE
"UnsupportedOperation: fileno" in Colaboratory

### ISSUE THREAD

**Issue by ChrisCummins** [facebookresearch/CompilerGym#10]
## 🐛 Bug

`UnsupportedOperation: fileno` error is raised on environment creation in Google Colaboratory.

## To Reproduce

![image](https://user-images.githubusercontent.com/1636663/102848878-cd9fd600-440d-11eb-9f87-fd0cddf66e67.png)

## Environment

Please fill in this checklist:

- CompilerGym: v0.1.0
- How you installed CompilerGym (conda, pip, source): pip
- OS: Colab hosted runtime

**Comment by ChrisCummins** [comment]
Unable to reproduce locally in ipython:

```py
$ ipython
Python 3.8.5 (default, Sep  4 2020, 02:22:02)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import compiler_gym

In [2]: import gym

In [3]: env = gym.make("llvm-v0")

In [4]: env.reset()
```

Cheers,
Chris

**Comment by ChrisCummins** [comment]
Also unable to reproduce locally using jupyterlab:

```sh
$ conda create -n jupyter python=3.8
$ conda activate jupyter
$ conda install -c conda-forge jupyterlab
$ python -m ipykernel install --user --name myenv --display-name "Python (conda)"
$ jupyter-lab
```

![image](https://user-images.githubusercontent.com/1636663/103228017-c0ac4680-4927-11eb-9d73-3bf04fe86f26.png)

**Comment by ChrisCummins** [comment]
I found that the error is caused by attempting to pipe `subprocess.Popen()` output to `sys.stderr`:

![image](https://user-images.githubusercontent.com/1636663/103228223-27316480-4928-11eb-9195-5431a8f7da48.png)

In colab, `sys.stderr` is a `ipykernel.iostream.OutStream` (same in Jupter lab).
